src/docs/tutorial-extended-data/filter-with-extended-data/index.md

```diff
@@ -3,7 +3,7 @@ title: Filter listings with extended data
 slug: filter-with-extended-data
 updated: 2020-07-14
 category: tutorial-extended-data
-ingress: Learn how to filter listings using public data key.
+ingress: Make the most of extended data and learn how to filter and query listings using the public data key.
```

Comment:
I guess we could drop the "key" here. What do you think?

Reply:
It does sound better, but dropping key would drop the length by 6 characters and in terms of SEO the more length we have here, the better 🤔

Reply:
OK, I see. How about: _Make the most of extended data and learn how to use listing public data as a filtering query parameter in listings search._